renku/api/datasets.py

```diff
@@ -46,11 +47,19 @@ class DatasetsApiMixin(object):
     DATASETS = 'datasets'
     """Directory for storing dataset metadata in Renku."""
 
+    REFS_DATASETS = 'refs'
```

Comment:
why do you need it? the `LinkReference` object should provide you with enough abstraction.